artifactory/commands/buildinfo/publish_test.go

```diff
@@ -67,3 +72,34 @@ func TestExcludePasswordsPattern(t *testing.T) {
 		t.Error("expected:", expected, "got:", filteredKeys)
 	}
 }
+
+func TestPrintBuildInfoLink(t *testing.T) {
+	var linkTypes = []struct {
+		artVersion    int
```

Comment:
Similarly to my previous comment, the name ```artVersion``` is wrong. Should be ```majorVersion```.
Let's also add a test for getting the majorVersion out of a real version string.